ts/session/apis/open_group_api/sogsv3/sogsV3AddRemoveMods.ts

```diff
@@ -0,0 +1,68 @@
+/** MODERATORS ADD/REMOVE */
+
+import AbortController from 'abort-controller';
+import { PubKey } from '../../../types';
+import { OpenGroupRequestCommonType } from '../opengroupV2/ApiUtil';
+import { batchFirstSubIsSuccess, sogsBatchSend } from './sogsV3BatchPoll';
+
+/**
+ * Add those pubkeys as admins.
+ * We do not support adding as moderators/visible/global for now in session desktop
```

Comment:
Maybe put a TODO here so it's easily found in future?

Reply:
added TODO